FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Basic_TLS/mqtt_broker_setup.txt

```diff
@@ -0,0 +1,57 @@
+It is our recommendation to always use strong mutual authentication in any Internet of Things
+application. Instructions below are for setting up a local Mosquitto broker that supports
+TLS server-only authentication for use with this MQTT demo.
+1. Generate certificates with OpenSSL.
+    a. Download and install [Git For Windows](https://git-scm.com/download/win).
+    b. Open PowerShell and enter the following commands to generate TLS certificates:
+      i.    cd "C:\Program Files\Git\usr\bin" # If Git is installed elsewhere, update the path.
+      ii.   mkdir $home\Documents\certs
+      iii.  .\openssl.exe req -x509 -nodes -sha256 -days 365 -newkey rsa:2048 -keyout $home\Documents\certs\ca.key -out $home\Documents\certs\ca.crt
```

Comment:
is openssl.exe locally availably in the directory C:\Program Files\Git\usr\bin?

Reply:
Yes, it's always there unless Git is installed elsewhere.